roles/wazo_certs/tasks/main.yml

```
@@ -0,0 +1,42 @@
+---
+- name: Install xivo-certs
+  apt:
+    name: xivo-certs
+    state: latest
+
+- name: Install custom certificate for HTTPS
+  copy:
+    src: "{{ engine_api_https_cert }}"
+    dest: /usr/share/xivo-certs/server.crt
+    owner: root
+    group: www-data
+    mode: 0640
+  when: engine_api_https_cert is defined
```

Later on we should check those roles with Molecule. Which is picky about idempotence. Hence don't install those certificates if you're going to remove them in the same play.

```suggestion
  when: 
    - engine_api_https_cert is defined
    - runtime
```